File: Desenvolvimento Back-End PHP/03/poo/Cliente.php
<?php
    class Cliente {
        
        private string $_cpf;
        private string $_nome;
        private string $_telefone;
        private string $_endereco;
        
        function __construct($cpf, $nome, $telefone, $endereco)
        {
            $this->_cpf = $cpf;
            $this->_nome = $nome;
            $this->_telefone = $telefone;
            $this->_endereco = $endereco;
        }

        public ContaPoupanca $contaPoupanca;

        function VisualizarRelatorio() {
            $relatorio = PHP_EOL . 
                "*********************************" . PHP_EOL . 
                "A conta de numero";
        }

    }
?>

File: Outros/POO UML/Codigos/Aula 1/Exemplo banco/c-sharp/poo_conta_01/Cliente/Cliente.cs
using poo_conta.Servicos;

namespace poo_conta.Cliente
{
    public class Cliente
    {
        public string CPF { get; set; }
        public string Nome { get; set; }
        
        public string Telefone { get; set; }
        
        public string Endereco { get; set; }
        
        // Faz a relação entre a classe Cliente e a conta poupança
        public ContaPoupanca ContaPoupanca { get; set; }
                
        public void VisualizarDetalhesCliente()
        {
            string relatorio = 
                $"\n\n******************************************************" +
                $"\nA conta de número: {ContaPoupanca.NumeroConta} " + 
                $"de agência {ContaPoupanca.Agencia}, pertencentes ao cliente de nome\n" +
                $"{Nome.ToUpper()} e de CPF: {CPF}\n" +
                $"localizado no endereço {Endereco} e de telefone {Telefone} foi criado" + 
                $"com sucesso!\n" +
                $"\n******************************************************\n";

            System.Console.WriteLine(relatorio);
        }

    }
}